schema/herocard-response-schema.json

```diff
@@ -95,6 +95,11 @@
           "items": {"type": "object"},
           "description": "The content of this field, which can consist of arbitrary JSON objects. The client will use the field's type attribute to determine how to parse the content."
         },
+        "comments": {
+          "type": "array",
+          "items": {"$ref": "#/definitions/cardBodyFieldCommentsItem"},
+          "description": "It enables users to view all the comments added"
+        },
```

Comment:
Are we going to want to consider versioning for these schemas at any point?

Reply:
Don't think we have added versioning in any of the APIs. Its backward compatible, not sure we need it right now.